src/Api/SecretsManager/Controllers/AccessPoliciesController.cs

```diff
@@ -336,4 +341,39 @@ await _accessPolicyRepository.GetServiceAccountGrantedPoliciesPermissionDetailsA
                 userId, accessClient);
         return new ServiceAccountGrantedPoliciesPermissionDetailsResponseModel(results);
     }
+
+    public async Task LogAccessPolicyServiceAccountChanges(IEnumerable<BaseAccessPolicy> currentPolicies, IEnumerable<BaseAccessPolicy> updatedPolicies, Guid userId)
+    {
+        foreach (var current in currentPolicies.OfType<GroupServiceAccountAccessPolicy>())
+        {
+            if (updatedPolicies.Where(r => r.Id == current.Id).Count() <= 0)
```

Comment:
Thsi could be changed into `.Count( id == id)` or a `!...Any(id == id)`.

It would get rid of the warning messages.